app/src/forms/email/emailService.js

```diff
@@ -204,7 +204,7 @@ const service = {
     try {
       const form = await formService.readForm(formId);
       const submission = await formService.readSubmission(submissionId);
-      const bodyTemplate = form.identityProviders[0].idp.includes('public') ? 'submission-received-confirmation-public.html' : 'submission-received-confirmation-login.html';
+      const bodyTemplate = form.identityProviders.length > 0 && form.identityProviders[0].idp.includes('public') ? 'submission-received-confirmation-public.html' : 'submission-received-confirmation-login.html';
```

Comment:
`.idp` is a string or another array? If it's a string it wouldn't be the includes check would it?

Includes would be for if you were checking the `.identityProvider` array for at least 1 object that has an `idp === 'public'